src/libs/actions/Policy/Category.ts

```diff
@@ -266,6 +298,18 @@ function createPolicyCategory(policyID: string, categoryName: string) {
     API.write(WRITE_COMMANDS.CREATE_WORKSPACE_CATEGORIES, parameters, onyxData);
 }
 
+function importPolicyCategories(policyID: string, categories: PolicyCategory[]) {
+    const onyxData = updateImportSpreadsheetData(categories.length);
+
+    const parameters = {
+        policyID,
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        categories: JSON.stringify([...categories.map((category) => ({name: category.name, enabled: category.enabled, 'GL Code': category['GL Code']}))]),
```

Comment:
Let's ensure the GL Code is a string, either here or when reading the file.

```suggestion
        categories: JSON.stringify([...categories.map((category) => ({name: category.name, enabled: category.enabled, 'GL Code': String(category['GL Code'])}))]),
```